packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -273,6 +294,8 @@ class InputDecoration {
       prefixStyle,
       suffixText,
       suffixStyle,
+      counterText,
+      counterStyle,
```

Comment:
at some point soon we're really going to have to turn this into a Diagnosticable